runtime/util/ConfigurationSerializer.cpp

```diff
@@ -665,6 +659,77 @@ void write_string_to_proof_trace(void *proof_writer, char const *str) {
       ->write_null_terminated_string(str);
 }
 
+void serialize_configuration_to_proof_trace(
+    FILE *file, block *subject, uint32_t sort) {
```

Comment:
Is this `FILE *file` right?

Reply:
Why not `void *proof_writer`?

Reply:
This method now just writes to a file. Before it needed a pointer to the abstract class but now the abstraction is not at the level where this function will be called. When this is called we know we want to write to a file so that is why I have changed back to accept a file pointer as it was before I introduced the lower level abstract writer.

Reply:
Perfect! Thanks for the explanation!